core-services/prow/02_config/maistra/maistra-test-tool/_prowconfig.yaml

```diff
@@ -0,0 +1,12 @@
+tide:
+  queries:
+  - labels:
+    - auto-merge
+    - okay to merge
```

Comment:
should we quote this string as in https://github.com/maistra/test-infra/blob/7de39fa91c919a4f89d09b8efe018777c83fff70/prow/config/tide.yaml#L41 ?

Reply:
@jwendell The `make update` target overwrites my quote around okay to merge . 
That's why I can't quote that string in this OpenShift CI PR.